backend/src/main/java/com/elshop/backend/common/FakerUtils.java

```diff
@@ -0,0 +1,43 @@
+package com.elshop.backend.common;
+
+import com.elshop.backend.product.model.Product;
+import com.github.javafaker.Faker;
+
+import java.util.Arrays;
+import java.util.List;
+
+public class FakerUtils {
+
+    private static final Faker faker = new Faker();
+
+    private static final AppUtils appUtils = new AppUtils();
+
+    private FakerUtils() {
+    }
+
+    public static Product generateFakeProduct(int imagesNr) {
+        List<String> images = generateImagesList(imagesNr);
+        String id = appUtils.generateUuid();
+        String title = faker.commerce().productName();
+        String slug = appUtils.generateSlug(title);
+
+        double price = Double.parseDouble(faker.commerce().price(1, 1000));
```

Comment:
Double, Float, etc. sind Zahlen im Binärformat. Bitte unbedingt BigDecimal verwenden, um Ergebnisse zu erzielen, die wir als Mensch (im Zehnersystem) auch erwarten würden.
```suggestion
        BigDecimal price = faker.commerce().price(1, 1000);
```